tests/general/URLBearTest.py

```diff
@@ -37,6 +37,16 @@ def test_detect_url_result(self):
                          [3, 'http://www.google.com/404',
                           LINK_CONTEXT.no_context])
 
+    def test_detect_pip_vcs_url_result(self):
+        valid_file = """
+        git+http://www.github.com/foo
```

Comment:
Just a nitpicky comment. `git+http` pip vcs url usually has a `.git` suffix in it ;)